tests/js/spec/views/organizationIntegrations/integrationRow.spec.jsx

```diff
@@ -0,0 +1,164 @@
+        <IntegrationRow
+          organization={org}
+          type="firstParty"
+          slug="bitbucket"
+          displayName="Bitbucket"
+          status="Installed"
+          publishStatus="published"
+          configurations={1}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('bitbucket');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Bitbucket');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/integrations/bitbucket/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').props().children).toEqual('1 Configuration');
+    });
+
+    it('has not been installed', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="firstParty"
+          slug="github"
+          displayName="Github"
+          status="Not Installed"
+          publishStatus="published"
+          configurations={0}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('github');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Github');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/integrations/github/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Not Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').exists()).toEqual(false);
+    });
+  });
+  describe('Plugin', function() {
+    it('has been installed', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
+          type="plugin"
+          slug="twilio"
+          displayName="Twilio (SMS) "
+          status="Installed"
+          publishStatus="published"
+          configurations={1}
+          data-test-id="integration-row"
+        />,
+        routerContext
+      );
+      expect(wrapper.find('PluginIcon').props().pluginId).toEqual('twilio');
+      expect(wrapper.find('IntegrationName').props().children).toEqual('Twilio (SMS) ');
+      expect(wrapper.find('IntegrationName').props().to).toEqual(
+        `/settings/${org.slug}/plugins/twilio/`
+      );
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+      expect(wrapper.find('PublishStatus').exists()).toEqual(false);
+      expect(wrapper.find('StyledLink').props().children).toEqual('1 Configuration');
+    });
+
+    it('has not been installed', async function() {
+      const wrapper = mountWithTheme(
+        <IntegrationRow
+          organization={org}
```

Comment:
@NisanthanNanthakumar We should have a test with >1 configurations. Also, we should check the configuration count that's visible matches the prop value.